### PR TITLE
Update pagination.rst

### PR DESCRIPTION
**Pull request by MarwanSalim**
Added more code samples to cater for out-of-range issue

### CODE REVIEW

**Comment by dereuromark** [comment]
An alternative was a custom PaginatorComponent
https://gist.github.com/dereuromark/46e039e793d0f3e06925e92da3d92145
Not sure if that is still feasable in 4.x/5.x

**Comment by dereuromark** [comment]
fi t

File: en/controllers/pagination.rst [hunk 1]
```diff
@@ -283,6 +283,17 @@ block and take appropriate action when a ``NotFoundException`` is caught::
         }
     }
 
+When the page is out of range, the query will be empty. Other possible solution is to check if there is an empty query
+and the selected page is greater than 1, unset the page and redirect to the first page::
```
Comment (markstory) [inline en/controllers/pagination.rst:287]:
```suggestion
When an out of range page is requested, the query will be empty. You can check if the page query
is empty and the selected page is greater than 1, and unset the page and redirect to the first page::
```

**Comment by othercorey** [comment]
Should this target 4.x not 4.next?

**Comment by MarwanSalim** [comment]
> Should this target 4.x not 4.next?

Resolved. Thank you

**Comment by MarwanSalim** [comment]
I'm sorry, further study is needed for this alternative solution. I closed it as of now.

**Pull request closed without merging**